src/LineBot/LineBot.csproj

```diff
@@ -43,4 +43,13 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.2" />
   </ItemGroup>
+
```

Comment:
Can you revert the changes to this file?

Reply:
I tried Reverting the File and 
![image](https://user-images.githubusercontent.com/33629087/48377369-b87df680-e721-11e8-9fc7-a2b867969949.png)